templates/docs/troubleshooting.md

```diff
@@ -10,3 +10,14 @@ $ go get -u github.com/motemen/gore
 
 For information see [https://github.com/gobuffalo/buffalo/issues/108](https://github.com/gobuffalo/buffalo/issues/108) and [https://github.com/motemen/gore/issues/63](https://github.com/motemen/gore/issues/63).
 {{/panel}}
+
+{{#panel name="buffalo dev fails to start"}}
```

Comment:
Sorry, my bad, I didn't explain myself. :) keep the `title` attribute but add a `name`. something like `name="dev-fails"`.